### PR TITLE
Better CI configuration

### LINKED ISSUE #213: Add golangci-lint reviews/comments via reviewdog

### PR DESCRIPTION
**Pull request by AlekSi**
Closes #213.

### CODE REVIEW

File: tools/tools.go [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// +build tools
+
+package tools // import "gopkg.in/reform.v1/tools"
+
+import (
+	_ "github.com/AlekSi/gocoverutil"
+	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
+	_ "github.com/quasilyte/go-consistent"
+	_ "github.com/reviewdog/reviewdog/cmd/reviewdog"
+	_ "golang.org/x/tools/cmd/goimports"
```
Comment (rumyantseva) [inline tools/tools.go:10]: So, you specifically define some tools for limiting, but nor all of them (e.g. not depguard), why?

Comment (AlekSi) [reply tools/tools.go:10]: Tools here are commands that will be build into `bin/`. depguard and other dependencies are already have versions defined: https://github.com/golangci/golangci-lint/blob/master/go.mod

File: .golangci-required.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+linters:
+  disable-all: true
+  enable:
+    - deadcode
+    - depguard
+    - errcheck
+    - goimports
+    - golint
+    - gosec
```
Comment (rumyantseva) [inline .golangci-required.yml:24]: Ok, in the main `.golangci.yml` you exclude some checks for gosec. Here you use the default config for it. Why?

File: tools/tools.go (hunk 1 above)
Comment (rumyantseva) [reply tools/tools.go:10]: Ah, got it! Thanks!

**Pull request merged**